tests/js/client/replication2/replication2-cluster.js

```diff
@@ -39,8 +40,8 @@ const {
 } = jsunity.jsUnity.assertions;
 
 // The RestLogHandler is only available in maintainer mode.
-const isInMaintainerMode = require('internal').db._version(true)['maintainer-mode'] === 'true';
-const replication2Enabled = require('internal').db._version(true)['replication2-enabled'] === 'true';
+const isInMaintainerMode = require('internal').db._version(true).details['maintainer-mode'] === 'true';
```

Comment:
Note: this also changes the access to the result of `db._version(true)` on the client side, i.e. in arangosh, but this PR did not modify the result structure of client-side version info.
I guess this change is made here is still ok because the previous access here was just wrong.

Reply:
Correct! The reason I found this inconsistency in the first place was because I copied this code to use somewhere else, but it did not work as expected. Due to this error these tests were always skipped before.